README.md

```diff
@@ -195,6 +292,10 @@ programming resources.
   should naturally reside in the model).
 <sup>[[link](#skinny-controllers)]</sup>
 
+* <a name="controller-LOC"></a>
+  Each controller action should (ideally) be under 6 lines of code.
```

Comment:
the ideal controller action is 1 line.  Up to 6 may be necessary for a create or update with error handling.  Include the examples from the doc here please.